crates/external_services/src/grpc_client/unified_connector_service.rs

```
@@ -378,6 +384,14 @@ pub fn build_unified_connector_service_grpc_headers(
             parse("api_secret", api_secret.peek())?,
         );
     }
+    if let Some(auth_key_map) = meta.auth_key_map {
+        let auth_key_map_str = serde_json::to_string(&auth_key_map)
+            .map_err(|_| UnifiedConnectorServiceError::ParsingFailed)?;
```

Why the error is discarded here? do change context

---

now I have logged the error.

change_context cannot be used here because the function returns Result<MetadataMap, UnifiedConnectorServiceError> (std::result::Result), not error_stack::Result